### ISSUE TITLE
laws of thermodynamics

### ISSUE THREAD

**Issue by nilis24** [nilis24/physics.h#9]
Is your feature request related to a problem? Please describe.
computationally calculate the laws of thermodynamics in C

Describe the solution you'd like
Code the laws of electricity in package/thermodynamics/thermodynamics.h

**Comment by nilis24** [comment]
@saiteja1290 do you want this issue to work on?

**Comment by saiteja1290** [comment]
Yeah Sure , Can you explain how do you want this to be done , like what formulas to be added.?

**Comment by nilis24** [comment]
Okay, assigned. I want the laws of thermodynamics as functions in C. you have to create a directory in ./package called "thermodynamics" and then inside of this, create a file called "thermodynamics.h" and put your code there. Then, in physics.h include the "thermodynamics.h" file

**Comment by saiteja1290** [comment]
Thank you , will submit the PR soon

**Comment by varunnitian** [comment]
I would also like to work on thermodynamics. Can you assign it to me also?

**Comment by nilis24** [comment]
@varunnitian no, only one person for each issue, sorry

**Comment by saiteja1290** [comment]
@nilis24 , hey I saw that there is no separate directory for Thermodynamics , do you want me to add one and then make the changes.?
